tests/integration/test_boto3.py

```diff
@@ -23,6 +47,19 @@ def test_boto_stubs(tmpdir):
         VerifiedHTTPSConnection('hostname.does.not.matter')
 
 
+@boto3_vendored
+def test_boto3_awsrequest_stubs(tmpdir):
+    with vcr.use_cassette(str(tmpdir.join('boto3-stubs.yml'))):
+        from botocore.awsrequest import AWSHTTPConnection, AWSHTTPSConnection
+        from vcr.stubs.boto3_stubs import VCRRequestsHTTPConnection, VCRRequestsHTTPSConnection
+        # addopted from test_boto_vendored_stubs, but
+        # Todo this seems to be other way around
+        assert issubclass(AWSHTTPConnection, VCRRequestsHTTPConnection)
```

Comment:
I wonder if the logic should not be other way around? (also in fn above)
issubclass(VCRRequestsHTTPConnection. AWSHTTPConnection)